lib/sedna/algorithms/multi_task_learning/multi_task_learning.py

```diff
@@ -68,9 +66,7 @@ def __init__(self,
         self.extractor = None
         self.base_model = estimator
         self.task_groups = None
-        self.task_index_url = Context.get_parameters(
-            "MODEL_URLS", '/tmp/index.pkl'
-        )
+        self.task_index_url = "index.pkl"
         self.min_train_sample = int(Context.get_parameters(
             "MIN_TRAIN_SAMPLE", '10'
         ))
```

Comment:
put  `MIN_TRAIN_SAMPLE`  to contant.py.
set `10` to a contant

Reply:
fix